mosaic_api/mosaic_api/__init__.py

```diff
@@ -0,0 +1,5 @@
+"""mosaic_apiboard_api."""
+
+import pkg_resources
+
+version = pkg_resources.get_distribution(__package__).version
```

Comment:
we should avoid this, it makes startup really slow. 

We switched to hardcoded version in most of out lib (and use https://github.com/c4urself/bump2version to update versions accros the files)